pages/design/rule-template.md

```diff
@@ -57,90 +60,94 @@ _There are no major accessibility support issues known for this rule._
 
 ### Passed
 
-#### Passed example 1
+#### Passed Example 1
 
 Description...
 
 ```html
 <!-- code -->
 ```
 
-#### Passed example 2
+#### Passed Example 2
 
 ...
 
 ### Failed
 
-#### Failed example 1
+#### Failed Example 1
 
 Description...
 
 ```html
 <!-- code -->
 ```
 
-#### Failed example 2
+#### Failed Example 2
 
 ...
 
 ### Inapplicable
 
-#### Inapplicable example 1
+#### Inapplicable Example 1
 
 Description...
 
 ```html
 <!-- code -->
 ```
 
-#### Inapplicable example 2
+#### Inapplicable Example 2
 
 ...
 ````
 
-## Composite rules
+## Composite Rules
 
 Composite rules are rules that take results from different rules and through some logic come to a single result. For example: SC 1.2.3 allows video to pass with either a transcript, an audio description, or if it is a media alternative. Each of these would be atomic rules, that are used in a composite rule. The composite rule than describes that at least one of these must pass for the composite rule to pass.
 
 For more about composite rules, see the [ACT Rules Format](https://www.w3.org/TR/act-rules-format/#composed-rules). To create a composite rule, use the [empty composite rule template](./composite-template-empty.html).
 
-### Composite Rule template
+### Composite Rule Template
```

Comment:
Again: Would it be possible to embed the template file, so it doesn't have to maintained in two different places?
They have previously been a bit out of sync, which is confusing.

Reply:
Probably, that seems separate from this pull request.